src/main/java/com/minecolonies/client/gui/WindowTownHall.java

```diff
@@ -319,6 +483,59 @@ public void updateElement(int index, Pane rowPane)
         {
             findPaneOfTypeByID("toggleJob", Button.class).setLabel(LanguageHandler.format("com.minecolonies.gui.hiring.on"));
         }
+
+        ScrollingList workOrderList =  findPaneOfTypeByID(LIST_WORKORDER, ScrollingList.class);
+        workOrderList.enable();
+        workOrderList.show();
+        //Creates a dataProvider for the unemployed citizenList.
+        workOrderList.setDataProvider(new ScrollingList.DataProvider()
+        {
+            /**
+             * The number of rows of the list.
+             * @return the number.
+             */
+            @Override
+            public int getElementCount()
+            {
+                return workOrders.size();
+            }
+
+            /**
+             * Inserts the elements into each row.
+             * @param index the index of the row/list element
+             * @param rowPane the parent Pane for the row, containing the elements to update
+             */
+            @Override
+            public void updateElement(int index, Pane rowPane)
+            {
+                final WorkOrderView workOrder = workOrders.get(index);
+                String claimingCitizen = "";
+
+                if(index ==0)
+                {
+                    rowPane.findPaneOfTypeByID("up",Button.class).hide();
+                }
+                else if(index == getElementCount()-1)
+                {
+                    rowPane.findPaneOfTypeByID("down",Button.class).hide();
+                }
+
+                //Searches citizen of id x
+                for(CitizenDataView citizen: citizens)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'citizen' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)